### ISSUE TITLE
Sound files

### ISSUE THREAD

**Issue by ThatMazes** [Razmoth/PGRStudio#3]
Good day, is it impossible to extract sound files from the game or are they  categorized under a different name?

**Comment by Razmoth** [comment]
Greetings,

Haven't looked into audio stuff with PGR, but it seems to be acb/awb criware files.

**Comment by RosettaYui** [comment]
I tried vgmtoolbox, but when I extracted acb/awb files, the acb ones sound strange，and the awb ones are exported to bin files.

**Comment by CleanerN** [comment]
> I tried vgmtoolbox, but when I extracted acb/awb files, the acb ones sound strange，and the awb ones are exported to bin files.

Hello. This is a month late but I found the solution to this. It's very easy to access the .awb + .acb and listen to them in their proper format. Use foobar2000 and the vgmstream plugin for fb2k. You do not need to convert the .awb or .acb into .bin. 

**FB2K Link**: https://www.foobar2000.org
**FB2K VGM Plugin Link**: https://dl.vgmstream.org

As of this global build of PGR with _Vera: Garnet_ and _The Last Spark_, the audio files are accessible this way. There appears to be a pattern in organization.

.acb appears to be audio for weapons, skills, and various attacks.
.awb appears to be audio for voicelines, and music.

I checked the CN server build with this method. It appears that they have taken more care into encrypting the files, as the current CN build produced .awb's and .acb's that only had static.
